### PR TITLE
FAQ: Address misconception re: multiple arguments

### PR DESCRIPTION
**Pull request by advayDev1**

### CODE REVIEW

**Comment by advayDev1** [comment]
I address -prefixes FILE because some people might think FILE is an "argument to the -prefixes argument"

File: FAQ.md [hunk 1]
```diff
@@ -9,6 +9,28 @@ You need at least [Gradle version 2.4](https://discuss.gradle.org/t/gradle-2-4-r
...
+
+    // ALSO CORRECT
+    translateArgs '-use-arc'
+    translateArgs '-prefixes'
+    translateArgs 'file.prefixes'
+
+    // WRONG
+    translateArgs '-use-arc -prefixes file.prefixes'
+    
+    // ALSO WRONG
```
Comment (confile) [inline FAQ.md]: // WRONG

File: FAQ.md [hunk 2]
```diff
@@ -9,6 +9,28 @@ You need at least [Gradle version 2.4](https://discuss.gradle.org/t/gradle-2-4-r
...
 
+### How do I properly pass multiple arguments to j2objc?
+
+Make sure your arguments are separate strings, not a single space-concatenated string.
+```
+j2objcConfig {
+    // CORRECT
+    translateArgs '-use-arc', '-prefixes', 'file.prefixes'
+
+    // ALSO CORRECT
```
Comment (confile) [inline FAQ.md]: // CORRECT

**Comment by confile** [comment]
see my comments then LGTM

**Comment by brunobowden** [comment]
LGTM

This should really be an error message as well. I'll work on that message when I do the wider changes for all the flags => args changes.

File: FAQ.md (hunk 1 above)
Comment (brunobowden) [inline FAQ.md]: I agree simplifying it to just CORRECT or WRONG

**Comment by advayDev1** [comment]
@brunobowden 
As I pointed out here: https://github.com/j2objc-contrib/j2objc-gradle/issues/203#issuecomment-114311174
this cannot be an error message

**Pull request merged**